### PR TITLE
chore: switch from standard to eslint + prettier

### PR DESCRIPTION
**Pull request by HashimotoYT**
The same what are we do in other repos. Here used `eslint + prettier` instead of single `prettier` for the ability to use rules.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (erickzhao) [inline .prettierrc:1]: I'm curious, do we have a standard ruleset that we're using across Electron-related projects?

Comment (malept) [reply .prettierrc:1]: Doesn't seem like it. For example, [`@electron/get`](https://github.com/electron/get/blob/master/.prettierrc.json).

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  },
+  extends: [
+    'prettier',
+    'plugin:prettier/recommended',
+  ],
+}
```
[the remark this answers is not in the document]
Comment (HashimotoYT) [reply .eslintrc.js:10]: I extracted that to own file since don't love littering `package.json` by these configs.

File: package.json [hunk 3]
```diff
@@ -10,15 +10,15 @@
   "scripts": {
     "start": "cross-env NODE_OPTIONS='--max_old_space_size=4096' NODE_PATH=. NODE_ENV=production node server.js",
     "bootstrap": "node script/bootstrap",
-    "test": "cross-env NODE_PATH=. NODE_ENV=test mocha --timeout 5000 && standard --fix",
+    "test": "cross-env NODE_PATH=. NODE_ENV=test mocha --timeout 5000 && eslint --ext .js --fix",
```
Comment (HashimotoYT) [inline package.json:13]:
```suggestion
    "test": "cross-env NODE_PATH=. NODE_ENV=test mocha --timeout 5000 && eslint . --fix",
```

**Pull request merged**